metadata-ingestion/src/datahub/ingestion/source/kafka_connect/sink_connectors.py

```diff
@@ -187,7 +187,7 @@ def get_parser(
         connector_manifest: ConnectorManifest,
     ) -> BQParser:
         project = connector_manifest.config["project"]
-        sanitizeTopics = connector_manifest.config.get("sanitizeTopics", "false")
+        sanitizeTopics = connector_manifest.config.get("sanitizeTopics") or "false"
```

Comment:
if `bool` type is expected for `sanitizeTopics`

```suggestion
        sanitizeTopics = connector_manifest.config.get("sanitizeTopics") or False
```

Reply:
the config defines this as string so keeping it string here. The actual boolean casting is done below :
` sanitizeTopics=sanitizeTopics.lower() == "true",`